pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -132,8 +132,10 @@ func defaultStrategyChooser(instance *v1alpha1.Jaeger) strategy.S {
 }
 
 func (r *ReconcileJaeger) apply(jaeger v1alpha1.Jaeger, str strategy.S) error {
-	if err := r.applyElasticsearches(jaeger, str.Elasticsearches()); err != nil {
-		return err
+	if strings.EqualFold(viper.GetString("es-provision"), v1alpha1.FlagProvisionElasticsearchTrue) {
+		if err := r.applyElasticsearches(jaeger, str.Elasticsearches()); err != nil {
```

Comment:
There should be a log message if strategy contains ES objects and ES API is not available.